File: feedbackhub.client/src/types/account/UserDetailDto.ts
export interface UserDetailDto {
    Id: number;
    Username: string;
    Email: string;
    Fullname: string;
    IsDeleted: boolean;
}

export interface ClientUserDetailDto extends UserDetailDto {
    Client: string;
    Applications: string[];
}

export interface UserProfileDto {
    Id: number;
    Username: string;
    Email: string;
    Fullname: string;
    Role : string;
    Client: string;
    AvatarBase64? : string;
    Applications : string[];
}

File: FeedbackHub.Domain/Dto/User/UserDetailDto.cs
namespace FeedbackHub.Domain.Dto.User
{
    public class UserDetailDto
    {
        public int Id { get; set; }
        public required string Fullname { get; set; }
        public required string Email { get; set; }
        public int? ClientId { get; set; }
        public string? Client { get; set; }
        public bool IsDeleted { get; set; }
        public List<string> Applications { get; set; } = new();
    }
}
